azure_arc_ml_jumpstart/aks/arm_template/aks.json

```diff
@@ -0,0 +1,150 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterName": {
+      "type": "string",
+      "defaultValue": "Arc-ML-AKS",
```

Comment:
Try to standardize on the "Demo" suffix and not the k8s distro.

Reply:
I'm not sure I'm following, this is directly in line with the [Data Services](https://github.com/microsoft/azure_arc/blob/53e89bbbfb96fbf2f231e1a321715ef81688f11d/azure_arc_data_jumpstart/aks/arm_template/aks.json#L7) scenarios, except 'Data' is replaced by 'ML`:
![image](https://user-images.githubusercontent.com/46581776/130327649-6c5057b6-b332-4fe1-8254-aa49ee8cd682.png)

Reply:
I am aware, this is something we need to fix across all scenarios so it will be generic and we won't have to worry about infra type.

Reply:
Makes sense, changing to `Arc-ML`